spec/features/user_deletes_book_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature 'User deletes a book', js: true do
+  scenario 'successfully' do
+    book = create(:book)
+    visit root_path
+
+    delete_book
+
+    expect(page).to have_content(t('books.index.header'))
+    expect(page).not_to have_content(book.title)
+    expect(page).not_to have_content(book.author)
```

Comment:
You can probably just test that the title is not there. If you can get away with less that usually makes test easier to maintain in the future